messaging/coap/engine.c

```diff
@@ -500,10 +512,14 @@ coap_receive_method_payload(coap_receive_ctx_t *ctx, const char *href,
   COAP_DBG("no block options; processing regular request");
   const uint8_t *incoming_block;
   uint32_t incoming_block_len = coap_get_payload(ctx->request, &incoming_block);
-  if (oc_drop_command(endpoint->device)) {
-    COAP_WRN("cannot process new request during closing TLS sessions");
+#ifdef OC_SECURITY
+  // Drop unsecured (unicast/multicast) requests during reset the device.
+  if (oc_reset_in_progress(endpoint->device) &&
+      ((endpoint->flags & SECURED) == 0)) {
```

Comment:
This drop messaging was introduced because `deviceID` will be changed after the reset. And second think is that the `ps.s == OC_DOS_RESET` so acls and other staff will also limit access to the device resources as was before.